### DISCUSSION TITLE
Got the Error "SSL Error: Certificate Authority is not trusted"

### DISCUSSION THREAD

**Discussion by sdinesmvc** [react-native-webview/react-native-webview#2333] [Q&A]
I have authorization endpoint refer below. it is hosted in virtual machine(accessible only intranet).
https://productdev.co:30200/user/connect/authorize?client_id=AMC-Cluster&redirect_uri=https://productdev.co:30200/apm/ui/signin-callback.html&response_type=token&scope=AMC-Cluster

When I got below error in Android Simulator (running command expo start) when include this in react-native-web-view(version: ^11.2.3). 
But not got this error, when I am accessing through browser inside simulator.

Encountered an error loading page, Object {
  "canGoBack": false,
  "canGoForward": false,
  "code": 3,
  "description": "SSL error: The certificate authority is not trusted",
  "loading": false,
  "target": 3,
  "title": "",
  "url": "https://productdev.co:30200/user/connect/authorize?client_id=AMC-Cluster&redirect_uri=https://productdev.co:30200/apm/ui/signin-callback.html&response_type=token&scope=AMC-Cluster",
}

**Comment by keithmaxx** [comment]
I've tried different variations of config for `network_security_config.xml` as suggested [in official Android docs](https://developer.android.com/training/articles/security-config#ConfigCustom), but nothing worked. Any new updates here?

**Comment by lucasjesusone** [comment]
I solve this problem as follow:

First of all, you need to init a new project react native: npx react-native init myApp

Inside this app, on app.tsx, you put the same code of expo

import React from 'react';
import { StyleSheet } from 'react-native';
import { WebView } from 'react-native-webview';

function App(): React.JSX.Element {

return (
<WebView
style={styles.safeArea}
source={{ uri: 'your-url.com' }}
/>
);
}

const styles = StyleSheet.create({
safeArea: {
flex: 1,
},
});

export default App;

you need to install:
npm install --save react-native-webview

and after that, go to:

node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewClient

inside this class, go to method onReceivedSslError and switch to:

@OverRide
public void onReceivedSslError(final WebView webView, final SslErrorHandler handler, final SslError error) {
handler.proceed();
}

and then, run react native again > npx react-native run-android
